lib/cosmos/gui/dialogs/tlm_details_dialog.rb

```diff
@@ -116,13 +116,6 @@ def initialize(parent, target_name, packet_name, item_name, packet = nil)
     end
 
     def value_update_timeout
-      # Check to see that the server is still running. If the user shut down
-      # the underlying tool (like PacketViewer for example) the $cmd_tlm_server
-      # will be nil.
-      unless $cmd_tlm_server
-        @timer.stop
```

Comment:
Don't think so.   If the below code keeps running it will just through DRbConnError which is caught and handled.